demos/llm_routing/README.md

```diff
@@ -0,0 +1,19 @@
+# LLM Routing
+This demo shows how you can arch gateway to manage keys and route to appropricate LLM.
```

Comment:
how are we routing right now?

Reply:
client can configure upstream llm based on `x-arch-llm-provider-hint` header

Reply:
see code change in chatbot-ui